Leuchtfeuer/mautic-MailjetAdapter-bundle
# Adding Send API transport

#### Description:

This pull request adds the new email transport based on the Email API v3.1.

#### Steps to test this PR:

1. Open this PR on Gitpod or pull it down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Clear the cache
3. Updated the DSN in `app/config/local.php` as `'mailer_dsn' => 'mautic+mailjet+api://<apiKey>:<secretKey>@default:465'`
4. Send the email. The email should be re-routed using Mailjet API.
